parsers/FR-COR_IT-SAR.py

```diff
@@ -0,0 +1,129 @@
+from datetime import datetime, timedelta
+from logging import Logger, getLogger
+from typing import Callable, Dict, List, Literal, Optional
+
+from requests import Session
+
+from .ENTSOE import fetch_exchange as fetch_ENTSOE_exchange
+from .ENTSOE import fetch_exchange_forecast as fetch_ENTSOE_exchange_forecast
+from .lib.config import refetch_frequency
+from .lib.exceptions import ParserException
+
+"""
+Fetches data for the IT-SACOAC->IT-SAR and IT-SACODC->IT-SAR exchanges
+by wrapping the ENTSOE.py parser and combining the data from the two
+to produce a single data list for the FR-COR->IT-SAR exchange.
+"""
+
+EXCHANGE_FUNCTION_MAP: Dict[str, Callable] = {
+    "exchange": fetch_ENTSOE_exchange,
+    "exchange_forecast": fetch_ENTSOE_exchange_forecast,
+}
+
+
+def fetch_data(
```

Comment:
Clean! 

I guess the best in theory would be to modify the ENTSO-E parser to accept multiple mappings, but as it's a much bigger project, we should only do that if we another case where that happens.

Reply:
I don't know how you guys will handle the historical data for aggregated region but one other motivation to add it to the ENTSO-E parser would be to fix the historical exchange between Sweden and NO-NO4.